.github/actions/compute-sha256/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'SHA256 of a file'
+description: 'Compute the SHA256 of a file'
+inputs:
+  path:
+    description: 'Path to a file.'
+    required: true
+outputs:
+  sha256:
+    description: 'The SHA256 of the file.'
+    value: "${{ steps.compute.outputs.sha256 }}"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Compute the sha256
+      id: compute
+      env:
+        UNTRUSTED_PATH: "${{ inputs.path }}"
+      run: |
+
+        set -euo pipefail
```

Comment:
I don't think we need this if we're just going to call out to another script. We can just do

```yaml
run: ./.github/actions/compute-sha256.sh
```

Reply:
yeah Ive been thinking about this too. I'm not yet 100% sure whether this action will be replaced/removed in the end. I was thinking that may be we will replace the script code with TS. 
But maybe it's better to use the script in this PR instead. Shall I do that?

Reply:
Let's go with the script option then.

Reply:
Well, let me know how you feel before I proceed :-)

Reply:
I was just referring to the `set -euo pipefail` part.

Reply:
sorry, I mis-understood. Gotcha.

Reply:
done, thanks